src/main/java/Sb2.java

```diff
@@ -23,24 +25,57 @@ public Sb2(String filePath) throws IOException {
 
         String pathToPackageJson = destPath + File.separator + "package.json";
         String jsonString = getFileContents(pathToPackageJson);
-        sb2Json = createJSONObject(jsonString);
+        init(createJSONObject(jsonString));
+    }
+    /**
+     * Construct an Sb2 object from a JSONObject. Useful for testing.
+     * @param stage JSONObject from which to construct Sb2
+     */
+    public Sb2(JSONObject stage) {
+        init(stage);
     }
     /**
+     * Function to be called from all constructors.
+     * @param stage JSONObject which is underlying data structure for Sb2.
+     */
+    public void init(JSONObject stage) {
```

Comment:
I don't like the name of this method. It's kind of hard to understand its role by reading it. Maybe `configureWithJson` or something?